manifests/bc.yaml

```diff
@@ -22,7 +22,54 @@ parameters:
     name: REPO_REF
     value: master
 objects:
-  - kind: "BuildConfig"
+  # use our own "gated" imagestream for the Jenkins master so we can test new
+  # images before upgrading
+  - apiVersion: v1
+    kind: ImageStream
+    metadata:
+      name: jenkins-s2i
+      namespace: fedora-coreos
+    spec:
+      tags:
+        - name: stable
+          from:
+            kind: DockerImage
+            # 2.138.2
+            name: openshift/jenkins-2-centos7@sha256:fd48b67ed2eca6c4042ac59dd3efd6763681bbc312b5f0c2a3adbfbd71f3c5ce
```

Comment:
- we lock on a specific commit here. we'll need to upgrade it in the future?
- should we specify the domain since it might move to quay.io in the future? I assume it's `docker.io` today?

Reply:
> we lock on a specific commit here. we'll need to upgrade it in the future?

Yeah, basically, because the master is so crucial, it feels like something we should update ourselves for now after sanity checking locally. (Or ideally by a bot later on when we have some CI in place).

> should we specify the domain since it might move to quay.io in the future? I assume it's docker.io today?

Yeah, good point. Will do.

Reply:
Fixed in latest rebased! :arrow_down: